client/state/reader/thumbnails/actions.js

```diff
@@ -48,7 +64,8 @@ export const requestThumbnail = ( embedUrl ) => ( dispatch ) => {
 			try {
 				return globalThis
 					.fetch( posterEndpoint )
-					.then( ( response ) => response.json() )
+					.then( handleFetchResponse )
+					.catch( () => {} )
```

Comment:
hmm I don't think it's right to put the catch block before the final `then`. Won't it swallow the error and then put the promise back into the resolved state and call the final `then( (json) => { } )` function?

I think this error swallowing catch should be at the end of the promise chain. And also IMHO it would be better to log an error to the console. Although I suppose a network error would be logged anyway 🤷

Reply:
Thanks @roo2 🙌 

Yeah, I agree about moving the `catch` at the end 👍 moved in 6c6e6efbf94577121cc8f14453975b7f25fdafed

As for logging another error, I don't think it will be necessary - a network error will be seen in the console anyway.